controllers/database/generate.go

```diff
@@ -0,0 +1,86 @@
+package database
+
+import (
+	"fmt"
+	"github.com/goharbor/harbor-cluster-operator/controllers/database/api"
+	pg "github.com/zalando/postgres-operator/pkg/apis/acid.zalan.do/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	databaseFailoversGVR = pg.SchemeGroupVersion.WithResource(pg.PostgresCRDResourcePlural)
+)
+
+// generatePostgreCR returns PostgreSqls CRs
+func (postgres *PostgreSQLReconciler) generatePostgresCR() (*unstructured.Unstructured, error) {
+	resource := postgres.GetPostgreResource()
+	replica := postgres.GetPostgreReplica()
+	storageSize := postgres.GetPostgreStorageSize()
+	version := postgres.GetPostgreVersion()
+	name := fmt.Sprintf("%s-%s", postgres.HarborCluster.Namespace, postgres.HarborCluster.Name)
+
+	conf := &api.Postgresql{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "postgresql",
+			APIVersion: "acid.zalan.do/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: postgres.HarborCluster.Namespace,
+			Labels:    postgres.Labels,
+		},
+		Spec: api.PostgresSpec{
+			Volume: api.Volume{
+				Size: storageSize,
+			},
+			TeamID:            postgres.HarborCluster.Namespace,
+			DockerImage:       "harbor.ymmoa.com/monitoring/postgresql:12",
```

Comment:
Why use this const value ?

Reply:
It's a test  image, I will remove this item.